Makefile

```diff
@@ -88,6 +88,12 @@ LOG=ANSIBLE_LOG_PATH="$(LOGFILE)"
 LOGINFO=$(INFO) "Wrote $(LOGFILE)"
 endif
 
+ifndef ANSIBLE_COLLECTIONS_PATH
+LINT_ANSIBLE_COLLECTIONS_PATH=${PWD}/collections:${HOME}/.ansible/collections:/usr/share/ansible/collections
```

Comment:
I don't think we want to include the non-local collection paths here. Including those was an oversight and a bug.